README.md

```diff
@@ -79,6 +79,18 @@ To quote (and sed) [prettier's page on option philosophy](https://prettier.io/do
 >
 > The more options ocamlformat has, the further from the above goal it gets. The debates over styles just turn into debates over which ocamlformat options to use.
 
+### How to locally disable ocamlformat?
+
+To disable the formatting of a specific expression you must attach an `[@@ocamlformat "option=VAL"]` attribute to this expression in the processed file, such as:
```

Comment:
This is technically attaching the attribute to the item. For an expression, it would be `[@ocamlformat ..]`

Reply:
Oh right, thanks